app/src/views/auth/Signin.vue

```diff
@@ -41,7 +42,8 @@ export default class Signin extends Mixins(LoadKeycloakRolesMixin, NrAffiliation
     // if there is stored NR data to process then affiliate it now
     const nr = JSON.parse(sessionStorage.getItem('NR_DATA'))
     if (nr) {
-      await this.createAffiliation(nr)
+      // Use the new "magic link routes" in the BRD to perform the affiliations and draft creations.
+      Navigate(this.magicLink(nr))
```

Comment:
I honestly don't know why this code is duplicated here (also in App.vue).

Can you test a bit with login scenarios and see which code is used, and then remove the unused code?

Reply:
I think this is fixed (subject to further testing).